peps/pep-0730.rst

```diff
@@ -63,6 +63,11 @@ significant platform differences between iOS and macOS.
 
 iOS code is compiled for compatibility against a minimum iOS version.
 
+Apple frequently refers to "iPadOS" in their marketing material. However, from a
+development perspective, there is no discernable difference between iPadOS and
+iOS. A binary that has been compiled for the ``iphoneos`` or ``iphonesimulator``
+ABIs can be deployed on iPad.
```

Comment:
It might be worth mentioning here that tvOS and watchOS are *not* included in this PEP.

Reply:
Updated to include a paragraph specifically excluding tvOS, watchOS and visionOS.